core/src/main/java/org/jdbi/v3/core/statement/OutParameters.java

```diff
@@ -14,17 +14,25 @@
 package org.jdbi.v3.core.statement;
 
 import java.sql.Date;
+import java.sql.ResultSet;
 import java.sql.Timestamp;
 import java.util.HashMap;
 import java.util.Map;
 
+import org.jdbi.v3.core.result.ResultBearing;
+
 /**
  * Represents output from a Call (CallableStatement).
  * @see Call
  */
 public class OutParameters {
+    private final StatementContext ctx;
     private final Map<Object, Object> map = new HashMap<>();
 
+    OutParameters(StatementContext ctx) {
```

Comment:
Adding this private constructor removes the existing default constructor, which technically makes this a breaking change. However, the constructor should never have been visible in the first place.

Can we add a compatibility comment to the release notes?